docs/reference/python_client.md

```diff
@@ -67,7 +67,7 @@ Example:
 ```python
 from lorax import Client
 
-endpoint_url = f"https://api.app.predibase.com/v1/llms/{llm_deployment_name}"
+endpoint_url = f"https://serving.app.predibase.com/{predibase_tenant_id}/llm/llm-deployment-{llm_deployment_name}/generate"
```

Comment:
Can we remove the `llm-deployment-` when resolving the URL in the backend - we would need to update the VS for this too.

Reply:
+1, less verbose the better.

Reply:
Can you also explain how to obtain the tenant ID?

Reply:
Also, we need to use `/` endpoint, not `/generate`.

Reply:
updated with information on the tenant short code. I'll need to update the VS to add `/` as a supported route